source/includes/fundamentals/code-snippets/CRUD/updateArray.go

```diff
@@ -86,13 +91,17 @@ func main() {
 
 	fmt.Println("\nPositional $[<identifier>] Operator:\n")
 	{
+		// Creates a filter and update document to match "sizes" array
+		// entries and remove those values
```

Comment:
S: just to make it a little clearer
```suggestion
		// values and remove those values
```